cache/lru_cache_test.cc

```diff
@@ -266,12 +266,13 @@ class TestSecondaryCache : public SecondaryCache {
   }
 
   std::unique_ptr<SecondaryCacheResultHandle> Lookup(
-      const Slice& key, const Cache::CreateCallback& create_cb,
-      bool /*wait*/) override {
+      const Slice& key, const Cache::CreateCallback& create_cb, bool /*wait*/,
+      bool& is_in_sec_cache) override {
```

Comment:
If we are sure we plan to let primary to decide whether it hopes the secondary cache to erase or not, keep in mind that we will soon add the argument. I'm not suggesting that we should add it now but keep in mind and you can decide when to add it.